Update BuildingTSMP2wCMake.md

joergbenke opened the pull request:
Hello,

I've corrected the compiler macros in the description of the build of TSMP2 with -DPARFLOW to -DParFlow because ParFlow in capital letters don't work (cmake complains).
I tested it with ParFlow standalone, but I corrected it also for the other occurences of -DPARFLOW.

joergbenke committed:
Update BuildingTSMP2wCMake.md

mvhulten commented on docs/users_guide/building_TSMP2/BuildingTSMP2wCMake.md:
Should this also be in camel case (for consistency)?

mvhulten committed:
Renamed another -DPARFLOW to -DParFlow

This covers all /-DPARFLOW\>/ in the tree; there are many extended /-DPARFLOW/
but that's ok.

mvhulten reviewed (approved):
Added additional 0260cb7b0c52f8fd97fa6f1dbc199b260bd50958 , fine now.
